Formula/voltdb.rb

```diff
@@ -14,11 +14,14 @@ class Voltdb < Formula
   end
 
   depends_on :ant => :build
+  depends_on "cmake" => :build
+
+  patch :DATA
```

Comment:
I'm sure they wont take this patch in it's current form (it would break linux distributions). Patch resolves incompatibilities between homebrew and their supported install targets (linux and download/extract).

Patch contains two fixes:
 - First change is basically same as inreplace above, but voltenv script differs a bit from other scripts so it's here.
 - Second change removes explict check in voltenv is voltdb installed behind symbolic links. 

Without patch voltenv gives error:
```
Warning: VoltDB exists in your path at:
    /usr/local/bin
  But this script is using VoltDB at:
    /usr/local/Cellar/voltdb/6.6/bin
```
I can file an issue and ask why there is such check. I can replace patch with inreplace commands.